website/docs/markdown-features.mdx

```diff
@@ -538,6 +538,27 @@ module.exports = {
 };
 ```
 
+If you want to add highlighting for languages not yet supported by Prism, you can swizzle `prism-include-languages` by
+
+```bash
+yarn swizzle @docusaurus/theme-classic prism-include-languages
+```
+
+It will produce `prism-include-languages.js` in your `src/theme` folder. You can add highlighting support for custom languages by editing `prism-include-languages.js`:
+
+```js {6} title="src/theme/prism-include-languages.js"
+    // ...
```

Comment:
Here we either need to remove all indents, or add a function declaration:

```js
const prismIncludeLanguages = (Prism) => {
    // ...

    additionalLanguages.forEach((lang) => {
      require(`prismjs/components/prism-${lang}`); // eslint-disable-line
    });

    require('/path/to/your/prism-language-definition');

    // ...
}

```

Reply:
@SamChou19815 have you missed  comment above or do you think this change is not necessary?

It seems important to me to provide some context if a piece of code is shown, otherwise its indents can be confusing for reader.

Reply:
@lex111  I had already added the function declaration in line 550.

Reply:
Ah, I'm sorry, I didn’t notice this update, because GitHub usually put the “Outdated” mark if the proposed change is applied.